src/config.ts

```diff
@@ -21,20 +21,26 @@ try {
 
 export const yamlConfig = _.merge(defaultConfig, customConfig)
 
-export class TransactionLimits implements ITransactionLimits {
-  readonly config
-  readonly level
-
-  constructor({ level }) {
-    this.config = yamlConfig.limits
-    this.level = level
+export const generateLimitConstants = (
+  limitsConfig = yamlConfig.limits,
+): LimitConstants => ({
+  oldEnoughForWithdrawalHours: limitsConfig.oldEnoughForWithdrawal / MS_IN_HOUR,
+  oldEnoughForWithdrawalMicroseconds: limitsConfig.oldEnoughForWithdrawal,
+})
+
+export const generateFeeConstants = (feesConfig = yamlConfig.fees): FeeConstants => ({
```

Comment:
this is a weird name "Generate Constants" :) maybe just use getFees or getFeeRates (although not sure about the use of withdrawFeeFlat)

Reply:
True. I was trying to convey `flat` vs `percentage` fees in the extra `withdrawFeeFlat` naming. What do we think about these instead:
- `depositFeeFixed`
- `depositFeeVariable`
- `withdrawalFeeFixed`
- `withdrawalFeeVariable`

With these new prop names I can also change `FeeConstants` to `FeeRates`.

**All proposed `Fee` changes reflected here: ...**

Reply:
unless I missed a recent changes, in our current implementation, deposits are `depositFeeVariable` and withdrawal are `withdrawalFeeFixed`

Reply:
Funny that we landed on the same names!

I didn't see that naming in my local version of things and couldn't find it in a Github search on here

![image](https://user-images.githubusercontent.com/17693119/127774879-ae52a331-daa3-4b2f-8e6c-22aeef2b7748.png)

Reply:
@vindard I think you must fix "Conflicting files" (also dont trust github indexation 😅)